### DISCUSSION TITLE
Supported PHP versions

### DISCUSSION THREAD

**Discussion by JBlond** [JBlond/php-diff#105] [General]
Interessting to see that most installs are using 7.4 see https://packagist.org/packages/jblond/php-diff/php-stats

**Comment by DigiLive** [comment]
I guess most developers/servers jumped from v7.2 to 7.4 and aren't ready for v8 yet.
As for me, I have a pretty large code base on v7.2 in a production environment. I'm stuck in between v7.2 and v8 😅

**Comment by vendeeglobe** [comment]
Is it to much work to maintain two branches, one with PHP 7 legacy support and one (master) using at least the latest PHP 8.0 features.

**Reply by DigiLive** [reply]
The library should run just fine in PHP 8.0.
What features do you need and why so?

**Reply by vendeeglobe** [reply]
It's less about necessity and more about capturing momentum. A new major PHP version is released and in the following months the new features are tested and discussed, tool sets are updated - in other words, it is present and the focus of interest and curiosity. I think this way the project has access to new PHP goodies sooner and can make better use of the interim.

**Comment by JBlond** [comment]
There are no new language features this lib could use for becoming a better software. At least I'm not aware of it.
